src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/INuGetProjectManagerService.cs

```diff
@@ -21,6 +21,17 @@ public interface INuGetProjectManagerService : IDisposable
         ValueTask<IInstalledAndTransitivePackages> GetInstalledAndTransitivePackagesAsync(
             IReadOnlyCollection<string> projectIds,
             CancellationToken cancellationToken);
+        /// <summary>
```

Comment:
NIT: Add an empty line.

Reply:
I will address this in a follow-up issue. https://github.com/NuGet/Home/issues/12098